nifi-registry-docker/dockerhub/DockerImage.txt

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apache/nifi-registry:0.1.0
```

Comment:
nifi-registry-0.1.0 is already released and master is now at nifi-registry-0.2.0-SNAPSHOT. I think this version should not be 0.1.0. Looking at what NiFi does, this should probably track towards the next version, i.e., `nifi-registry:0.2.0` (or perhaps `nifi-registry:0.2.0-SNAPSHOT`). @apiri do we have a convention for this file?

If one needs to publish a dockerhub image for NiFi Registry 0.1.0, that should probably be done by branching from the rel/nifi-registry-0.1.0 tag. Thoughts?

Reply:
@kevdoran I don't know if there's a convention, but looking at the changes to the equivalent NiFi file I think they do not use snapshot, just the base tag name.
This would also be more in line with how most Docker tags look